core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -72,46 +92,62 @@ private[spark] abstract class MemoryManager {
   def acquireUnrollMemory(
```

Comment:
Given that `acquireUnrollMemory` appears to act as a synonym for `acquireStorageMemory` in the current implementation, it might be worth adding a brief comment above this method to explain that this extra method exists in order to give us the future flexibility to account for unroll memory differently in the future.